week-3/slideshow/style.css

```diff
@@ -1 +1,93 @@
 /** Write your CSS in here **/
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  margin: 2rem 5rem 5rem 5rem;
+  display: grid;
+  justify-content: center;
+  background-image: url(./assets/back-ground.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow-x: hidden;
+  scroll-behavior: smooth;
+}
+
+h2 {
+  margin-bottom: 3rem;
+  text-align: center;
+  text-transform: uppercase;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 3rem;
+  text-shadow: 1.8px 2px 2px rgb(33, 11, 11);
+}
+
+#inputTime {
+  width: 100%;
+  height: 1.5rem;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  display: block;
+  outline: none;
+  border: none;
+  border-radius: 4px;
+  padding: 0.4rem;
+  transition: 0.3s;
+}
+
+#inputTime:focus{
+    border: 2.5px solid rgb(163, 159, 159);
+}
+
+#control-button {
+  border-radius: 0.2rem;
+  display: flex;
+  justify-content: center;
+  margin-top: 1rem;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#backward-btn,
```

Comment:
What is a better choice a css class or a css ID when we want to apply same styling across multiple elements?